prdoc/pr_6039.prdoc

```diff
@@ -0,0 +1,41 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: "Added Trusted Query API calls."
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Added is_trusted_reserve and is_trusted_teleporter API calls to all the runtimes.
+
+  - audience: Runtime User
+    description: |
+      Now you can query any runtime to check if pair<Asset, Location> is either a trusted reserve or trusted teleporter.
+
+crates:
+  - name: asset-hub-westend-runtime
+    bump: patch
```

Comment:
I think all of these should be `minor` because they're using features from a minor bump of another crate.
If you run `cargo update` (or something similar) all your crate versions will update to the latest `patch` bump, but not `minor` ones. This means that if someone updates their dependencies, they might break these crates since they depend on a `minor` bump of another crate.